core/new-gui/src/app/workspace/component/navigation/navigation.component.html

```diff
@@ -25,6 +25,9 @@
 
   </div>
   <div class="collapse navbar-collapse texera-navigation-execute">
+    <div class="button">
+      <button   class="btn btn-secondary texera-workspace-tour-run"  color="accent" [routerLink]="['/dashboard']">Dashbaord</button>
+    </div>
```

Comment:
this new button you just added should be parallel to `<div class="collapse navbar-collapse texera-navigation-execute">` instead of within it. You might create a new `div`, `<div class="collapse navbar-collapse texera-navigation-dashboard">` and put the new button within that `div`.